client/src/pages/component/Mission/Mission.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./Mission.css";
+
+function Mission() {
+	return (
+		<>
+			<div className="mission-constrainer">
+				<div className="mission-card">
+					<p className="mission-title">Our Mission</p>
```

Comment:
This is a heading, so do use a heading element. As this appears like the page's main heading, consider using `<h1>` here.